lib/hooks/gitlab/templates/note.html.haml

```diff
@@ -0,0 +1,41 @@
+:ruby
+  note                = payload.object_attributes
+  has_url             = note.url?
+  has_user            = payload.user?
+  repository_name     = has_url ? note.url.split('/', 5)[-2] : '' # http://example.com/owner/:name/issues/:id
+  has_issue           = payload.issue?
+  has_merge_request   = payload.merge_request?
+  has_snippet         = payload.snippet?
+  has_commit          = payload.commit?
+
+  if has_url
+    if has_issue
+      reference_id = payload.issue.iid
+      title = payload.issue.title
+    elsif has_merge_request
+      reference_id = payload.merge_request.iid
+      title = payload.merge_request.title
+    elsif has_snippet
+      reference_id = payload.snippet.id
+      title = payload.snippet.title  
+    elsif has_commit
+      reference_id = payload.commit.id
+    end
+  else
+    reference_id = note.id
+  end
+
+%p
+  - if has_user
+    %span= avatar_image_tag payload.user.avatar_url
+    = payload.user.name
+    commented on
+    = note.noteable_type
```

Comment:
Using `note.notable_type.humanize(capitalize: false)` will make the message natural.

Reply:
@ursm 
Thank you for review.　

After we are discussed ``note.noteable_type.underscore.humanize(capitalize: false)`` will make a more natural, because ``noteable_type`` have double word case like a ``MergeRequest``.

What do you think about it?
```
irb(main):002:0> 'MergeRequest'.humanize(capitalize: false)
=> "mergerequest"
irb(main):003:0> 'MergeRequest'.underscore.humanize(capitalize: false)
=> "merge request"
```

Reply:
Ah I see. I agree with your opinion.